frontend/src/components/SubResult.vue

```diff
@@ -17,111 +19,142 @@ function get_background_color(label: string | undefined) {
+        category,
+        assigned_labels
+      }))
+
+  entries.forEach((item) => {
+    if (item.assigned_labels && typeof item.assigned_labels === 'string') {
+      const validLabels = item.assigned_labels.split(',').filter(label =>
+        label !== 'niet van toepassing' &&
+        label !== 'nader te bepalen'
+      )
+
+      if (validLabels.length > 0) {
+        filtered[item.category] = validLabels
+      }
+    } else if (Array.isArray(item.assigned_labels)) {
+      const validLabels = item.assigned_labels.filter(label =>
+        label !== 'niet van toepassing' &&
+        label !== 'nader te bepalen'
+      )
+
+      if (validLabels.length > 0) {
+        filtered[item.category] = validLabels
+      }
+    }
+  })
+
+  return filtered
+})
+
+const hasLabels = computed(() => {
+  return Object.keys(filteredLabels.value).length > 0
+})
+
 </script>
 
 <template>
-  <div v-if="conclusion == ''">
-  <div class="rvo-accordion">
-    <details class="rvo-accordion__item">
-      <summary class="rvo-accordion__item-summary">
-        <h3
-          class="utrecht-heading-3 rvo-accordion__item-title rvo-heading--no-margins rvo-heading--mixed items-center"
-        >
-                  <span
-                    class="utrecht-icon rvo-icon rvo-icon-delta-omlaag rvo-icon--md rvo-icon--hemelblauw rvo-accordion__item-icon--closed"
-                    role="img"
-                    aria-label="Delta omlaag"
-                  ></span>
-          <span
-            class="utrecht-icon rvo-icon rvo-icon-delta-omhoog rvo-icon--md rvo-icon--hemelblauw rvo-accordion__item-icon--open"
-            role="img"
-            aria-label="Delta omhoog"
-          ></span>
-          {{ title }}
-        </h3>
-              <span class="rvo-accordion-teaser">Bekijk hier je AI Verordening profiel tot nu toe</span>
-      </summary>
-      <div class="rvo-accordion__content">
-        <div class="rvo-table--responsive py-5">
-          <table class="rvo-table">
-            <thead class="rvo-table-head">
-            <tr class="rvo-table-row">
-              <th scope="col" class="rvo-table-header">Categorie</th>
-              <th
-                scope="col"
-                class="rvo-table-header rvo-table-header"
-              >
-                Resultaat
-              </th>
-            </tr>
-            </thead>
-            <tbody class="rvo-table-body">
-            <tr v-for='(assigned_labels, category) in labels' :key="category" class="rvo-table-row">
-              <td class="rvo-table-cell rvo-text--bold rvo-text--md">{{ category }}</td>
-              <td class="flex rvo-table-cell gap-x-3">
-                <div v-for='(label) in assigned_labels' :key="label"
-                     class="rvo-tag rvo-tag--default" :style="get_background_color(label)">
-                  {{ label }}
-                </div>
-              </td>
-            </tr>
-            </tbody>
-          </table>
-        </div>
+  <div v-if="hasLabels">
```

Comment:
Actually there is no scenario where there is no labels right? Good to still implement it though, but even when you have no algorithm that is also a label

Reply:
There is 1 scenario: when the first question is not yet answered. In my opinion it looks weird to have an empty 'tussenresultaat' what do you think?